jdbc/src/test/java/com/interface21/jdbc/core/JdbcTemplateTest.java

```diff
@@ -1,5 +1,157 @@
 package com.interface21.jdbc.core;
 
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertDoesNotThrow;
+
+import java.sql.Connection;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.List;
+import javax.sql.DataSource;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
 class JdbcTemplateTest {
 
+    private final DataSource dataSource;
+    private final JdbcTemplate jdbcTemplate;
+
+    private static final RowMapper<User> userMapper = (resultSet) -> new User(
+            resultSet.getLong(1),
+            resultSet.getString(2),
+            resultSet.getString(3),
+            resultSet.getString(4)
+    );
+
+    @BeforeEach
+    void tearDown() {
```

Comment:
특정 update 행위에 대한 후속처리이므로 @AfterEach 어떤가요?

```suggestion
     @AfterEach
    void tearDown() {
```